Makefile

```diff
@@ -1,33 +1,37 @@
 .SHELLFLAGS += -e
 
 LINT_CONFIG=$(PWD)/lint-configs/tox.ini
+MODERN_LINT_CONFIG=$(PWD)/lint-configs/pyproject.toml
```

Comment:
What's the reason for not migrating everything to modern config?

Reply:
https://github.com/PyCQA/flake8/issues/234 I thought at this point everyone has heard of the shitstorm and drama.

Reply:
Heard of it but somehow forgot about it